verification/curator-service/api/src/controllers/access.ts

```diff
@@ -0,0 +1,20 @@
+import { NextFunction } from 'express';
+
+function setContainsAnyOf<T>(set: Set<T>, possibleElements: [T]): boolean {
```

Comment:
Instead of this function, intersection of two sets (roles and wantedRoles) can be:

`set1.filter(x => set2.has(x)));`